qutip/tests/solve/test_stochastic_me.py

```diff
@@ -92,6 +92,7 @@ def arccoth(x):
                         ['explicit1.5', 1e-4],
                         ['taylor2.0', 1e-4]]
     for n_method in list_methods_tol:
+        np.random.seed(1)
```

Comment:
Maybe the random seed should be outside the for loop and have a comment explaining why it is needed?